### ISSUE TITLE
example data not available

### ISSUE THREAD

**Issue by anuj2054** [mattrosen/rfSLAM#3]
in the example script inside the example folder, it references a dataset which is not available inside the repository.

After browsing through all the code and data, I do not see any example of how to use the functions for time varying covariates.

**Comment by azeamer** [comment]
Agreed. Very hard to implement without any example of data structure. 

Can the authors comment on when this might become available?
